### PR TITLE
fan-page 3 

### PR DESCRIPTION
**Pull request by brianandreasen1983**
Uploaded images for the fan-page. Removed some semantics in &nbsp for paragraphs. the &nbsp is still in for navigation as issues are occurring with the spacing right now waiting to change until more skills are obtained. 

Add some other css formatting as well.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
Submitted fan-page 4.

File: index-alt.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                <li>Race</li>
+                <li>Free-for-All</li>
+                <li>Domination (planned for future release)</li>
+
+            </ul>
+
+        </main>
+
+        <div id="explode">
+            <a href="images/screenshot00004.jpg"><img src="images/screenshot00004.jpg" height="100" width="100" alt="Robot fight"></a>
```
Comment (mbMosman) [inline index-alt.html]: Thumbnail link to larger image? :heavy_check_mark:

File: style-alt.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
+body {
+    font-family: didot;
+}
+
+h1 {
+    text-indent: 8em;
+    text-align: justify;
+    text-shadow: 2px 2px 4px rgba(68, 222, 31, 0.54);
+    font-family: garamond;
```
Comment (mbMosman) [inline style-alt.css]: you should provide fall back fonts for people who do not have the listed font.  See CSS Font Stack site for help building appropriate fallbacks.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+a {
+    color: red;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+ul {
+    list-style-type: georgian;
```
Comment (mbMosman) [inline style.css]: different list markers:  :heavy_check_mark:

**Comment by mbMosman** [comment]
Make sure to open another pull request for part 4 so I don't miss grading again.  :smiley:

**Pull request closed without merging**